### ISSUE TITLE
want detail of normalization in plt.hist(normed=True)

### ISSUE THREAD

**Issue by Aiamjay** [matplotlib/matplotlib#14213]
### Bug report

**Bug summary**
Actually, this is not a bug. I want to know some details or math behind the normalization in plt.hist. However, I cannot find any info in google or stackoverflow. So, could you please do me a favor?

**Code for reproduction**

```python
# Paste your code here
#
#
```

**Actual outcome**

```
# If applicable, paste the console output here
#
#
```

**Expected outcome**

**Matplotlib version**

  * Operating system:
  * Matplotlib version: 
  * Matplotlib backend (`print(matplotlib.get_backend())`):
  * Python version:
  * Jupyter version (if applicable):
  * Other libraries:

**Comment by Aiamjay** [comment]
At last, I've read your source code and found the answer. Just forget this issue.

**Comment by Aiamjay** [comment]
If ``True``, the first element of the return tuple will
            be the counts normalized to form a probability density, i.e.,
            the area (or integral) under the histogram will sum to 1.
            This is achieved by dividing the count by the number of
            observations times the bin width and not dividing by the total
            number of observations. If *stacked* is also ``True``, the sum of
            the histograms is normalized to 1.

**Comment by tacaswell** [comment]
Can you suggest a clarification to the docstring (https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.hist.html?highlight=hist#matplotlib.axes.Axes.hist ) or how we could have made this more discoverable?

**Comment by Aiamjay** [comment]
When I came across this problem, I found myself so confused by the detailed document provided by your organization. So, to be honest, I barely read all the docs and just want to find the answer quickly. Even when I told myself to read all of the information, I cannot find the answer. At last I read the source code. In the source code, your docs are really helpful. So, maybe, there isn't any thing wrong with your docstring. It's already good enough.
